src/System.Management.Automation/engine/interpreter/NumericConvertInstruction.cs

```diff
@@ -57,7 +57,7 @@ private object Convert(object obj)
                     case TypeCode.Byte: return ConvertInt32((Byte)obj);
                     case TypeCode.SByte: return ConvertInt32((SByte)obj);
                     case TypeCode.Int16: return ConvertInt32((Int16)obj);
-                    case TypeCode.Char: return ConvertInt32((Char)obj);
+                    case TypeCode.Char: return ConvertInt32((char)obj);
```

Comment:
Same here, better use official type names for all types here.

Reply:
```suggestion
                    case TypeCode.Char: return ConvertInt32((Char)obj);
```